configure.py

```diff
@@ -995,6 +995,11 @@
     default=None,
     help='Configure for clang-cl on Windows. This flag sets the GYP "clang" ' +
          'variable to 1 and "llvm_version" to the specified value.')
+parser.add_argument('--ccache-used',
```

Comment:
Normally we call the configure option `--use-something`. It's also probably better to scope it to Windows in case people think that it's needed on other platforms.

```suggestion
parser.add_argument('--use-ccache-win',
```